MusicStoreTests/ServicesTests/StatisticTests/AdminStatisticTests.cs

```diff
@@ -0,0 +1,144 @@
+﻿using Moq;
+using MusicStore.Business.Services.Statistics;
+using MusicStore.DataAccess;
+using MusicStore.DataAccess.Interfaces;
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace MusicStoreTests.ServicesTests.StatisticTests
+{
+    public class AdminStatisticTests
+    {
+        private readonly Mock<IUnitOfWork> mockUnitOfWork;
+        private readonly Mock<IRepository<BoughtSong>> mockBoughtSongRepository;
+
+        private const int DEFAULT_USER_ID = 1;
+
+        public AdminStatisticTests()
+        {
+            mockUnitOfWork = new Mock<IUnitOfWork>();
+            mockBoughtSongRepository = new Mock<IRepository<BoughtSong>>();
+        }
+        [Fact]
+        public void GetStatisticByTotalMoneyEarnedForSomeTimeTest()
```

Comment:
Add test for case, when user has no money spent. E.g. no money earned (collection empty)

Reply:
Provide wrong input data

Reply:
+